sk0-front/src/app/home/home.page.html

```diff
@@ -3,7 +3,7 @@
     <ion-row style="height: 100dvh" class="ion-align-items-center">
       <ion-col size="12" class="ion-text-center">
         <h1 class="ion-no-margin">Üdvözöllek kedves</h1>
```

Comment:
On the other hand we should come up with a rule, that we do not have strinliterals in the components (later we will need it for the internalizations). Emphasizing the "later" part.

Reply:
sure this is only to showcase how to make async pipe for an observable or behaviorsubject in this case